### PR TITLE
fix(common): FE-00 Ignore dependabot and merge commits

### PR DESCRIPTION
**Pull request by icatalina**
Implement a mechanism to ignore commits by author or committer and
ignore any commit from dependabot.

### CODE REVIEW

File: bin/validate-commits.js [hunk 1]
```diff
@@ -64,7 +73,6 @@ function getBranch() {
   if (process.env.TRAVIS_PULL_REQUEST && process.env.TRAVIS_PULL_REQUEST !== 'false') {
     return {
       from: process.env.TRAVIS_BRANCH,
-      to: process.env.TRAVIS_BRANCH,
```
Comment (icatalina) [inline bin/validate-commits.js:67]: This came from `${TRAVIS_BRANCH}..`, I'm guessing it is the `from` part 🤷 

definitely not `from` and `to`

**Comment by icatalina** [comment]
https://app.circleci.com/pipelines/github/bigcommerce/validate-commits/134/workflows/c3c4c27d-fe13-4d25-bb7b-2d4fb26ac897/jobs/138

Job with a fake dependabot commit, we can see it failing in master but not in the branch 👏

**Review by rtalvarez — APPROVED** [review]
🐑

**Review by jorgemoya — COMMENTED** [review]
Thanks!

**Pull request merged**